util/kube/kubetest/mock.go

```diff
@@ -19,29 +19,31 @@ type MockKubectlCmd struct {
 	APIResources []kube.APIResourceInfo
 	Commands     map[string]KubectlOutput
 	Events       chan watch.Event
+	LastValidate bool
```

Comment:
Guess I could have made this `*bool` so we could change it, but that seemed odd.